addons/website/static/src/xml/website.editor.xml

```diff
@@ -143,4 +143,12 @@
         <div class="mb-2" t-esc="contentText"/>
         <div class="o_ace_editor_container"/>
     </div>
+
+    <t t-name="website.new_content_loader">
+        <div class="o_new_content_loader_container position-fixed fixed-top fixed-left
+            h-100 w-100 d-flex flex-column align-items-center text-white font-weight-bold">
+            <p>Building your <span id="new_content_loader_module_name">website</span></p>
```

Comment:
- New term -> need pot file update for translation
- Dynamic term for a translation... bad idea to just randomly change a word inside.

The easiest way to do that properly: an empty `<p>` here in which you add a full sentence from the JS, translated via the JS using `_.str.sprintf(_t(...), ...)` like you did before.